src/Chainweb/Pact/RestAPI.hs

```diff
@@ -23,14 +20,7 @@ import Chainweb.Version
 -- -------------------------------------------------------------------------- --
 -- @GET /chainweb/<ApiVersion>/<ChainwebVersion>/chain/<ChainId>/pact/@
 
-type ApiV1API = SendApi :<|> PollApi :<|> ListenApi :<|> LocalApi
-
-type SendApi = "send" :> ReqBody '[JSON] SubmitBatch :> Post '[JSON] RequestKeys
-type PollApi = "poll" :> ReqBody '[JSON] Poll :> Post '[JSON] PollResponses
-type ListenApi = "listen" :> ReqBody '[JSON] ListenerRequest :> Post '[JSON] ApiResult
-type LocalApi = "local" :> ReqBody '[JSON] (Command Text) :> Post '[JSON] (CommandSuccess Value)
-
-type PactApi_ = "pact" :> ApiV1API
+type PactApi_ = "pact" :> API.ApiV1API -- TODO unify with Pact versioning
```

Comment:
We should really unify these. Pact api is `api/v1/send` etc. Chainweb is `[chainweb stuff]/pact/send`. Pact docs refer to the former. Do we fully conform, ie `[chainweb stuff]/pact/api/v1/send`? Or do we update the pact docs to start with `v1/send` and say the prefix can vary? Or, I guess we could modify `pact -s` to be `pact/v1/send` and then chainweb is strictly a prefix.

Reply:
Based on discussion with @larskuhtz , going with fully specified url, since the "Pact API" is cross platform while chainweb might hang other services off of `pact/` (ie SPV queries etc). Thus `[chainweb stuff]/pact/api/v1/send`.